cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -48,16 +48,16 @@ msgstr "Na, llenwais y ffurflen hon fy hun"
 
 #: cla_public/apps/base/constants.py:19
 msgid ""
-"I have difficulty using computers so someone filled in this form for me"
-msgstr "Rwyf yn cael trafferth defnyddio cyfrifiaduron, felly llenwodd rhywun arall y ffurflen hon ar fy rhan"
+"I had help filling in this form"
+msgstr ""
```

Comment:
Is there a reason the other two have kept their approximate translations but this one now has none? Was it decided that it was too different from "I had some other kind of help"?

Reply:
@exonian - good spot, that's an oversight on my part, will add it back in.